app/views/layouts/devise.html.erb

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html lang="<%= I18n.locale %>">
   <head>
-    <%= render "layouts/common_head", default_title: "Gobierno abierto" %>
+    <%= render "layouts/common_head", default_title: setting["org_name"] %>
```

Comment:
It would be great to have a test for this change so we don't accidentally remove it, for example when solving git conflicts :pray:.

How about adding:

```ruby
expect(page).to have_title "CONSUL"
```

To: https://github.com/consul/consul/blob/master/spec/features/registration_form_spec.rb#L38

If we want a better title, we could add:

```erb
<% provide(:title) { t("devise_views.users.registrations.success.title") } %>
```

To: https://github.com/consul/consul/blob/master/app/views/users/registrations/success.html.erb#L1

And then change the mentioned test to:

```ruby
expect(page).to have_title "Confirm your email address"
```

What do you think?

Reply:
Great suggestion